WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2328,6 +2340,45 @@ public void onMediaUploadCancelClicked(String localMediaId) {
     }
 
     @Override
+    public void onMediaDeleted(String localMediaId) {
+        if (!TextUtils.isEmpty(localMediaId)) {
```

Comment:
I don't understand why `localMediaId` is a string in several places when it should be an int, but as long as it's being passed around as a string we need to play it safe and catch any potential `NumberFormatException` if it fails to be parsed as an int. The simplest way to do that is use `StringUtils.stringToInt()`.